main.tf

```diff
@@ -2,25 +2,72 @@
 
 ## Exemplo
 
-# provider "google" {
-#   project = var.project
-#   region  = var.region
-# }
-
-# module "groundwork" {
-#   source   = "github.com/mentoriaiac/iac-modulo-groundwork.git?ref=v0.1.0"
-#   project  = var.project
-#   vpc_name = "groundwork"
-# }
-
-# variable "project" {
-#   description = "Nome do projeto (Default é staging)"
-#   type        = string
-#   default     = "direct-link-325016"
-# }
-
-# variable "region" {
-#   description = "Nome da região"
-#   type        = string
-#   default     = "us-central1"
-# }
\ No newline at end of file
+provider "google" {
+  project = var.project
+  region  = var.region
+}
+
+module "network_gcp" {
```

Comment:
Por que estamos usando esse módulo aqui? Eu acredito que o groundwork poderia estar em outro pipeline. Se for algo apenas pra agilizar, ok ;)

Reply:
Isso será resolvido como melhoria nesta Issue que foi aberta.
- https://github.com/mentoriaiac/iac-nomad/issues/7